src/AragonCourt.ts

```diff
@@ -89,57 +104,70 @@ export function handleModuleSet(event: ModuleSet): void {
     anj.decimals = anjContract.decimals()
     anj.save()
 
-    let config = CourtConfig.load(event.address.toHex())
     config.anjToken = anjAddress.toHex()
     config.save()
 
-    let registryModule = new JurorsRegistryModule(event.params.addr.toHex())
+    let registryModule = new JurorsRegistryModule(newModuleAdress.toHex())
     registryModule.court = event.address.toHex()
     registryModule.totalStaked = BigInt.fromI32(0)
     registryModule.totalActive = BigInt.fromI32(0)
     registryModule.totalDeactivation = BigInt.fromI32(0)
     registryModule.save()
   }
   else if (id == DISPUTE_MANAGER_ID) {
-    DisputeManager.create(event.params.addr)
+    DisputeManager.create(newModuleAdress)
     module.type = DISPUTE_MANAGER_TYPE
   }
   else if (id == VOTING_ID) {
-    Voting.create(event.params.addr)
+    Voting.create(newModuleAdress)
     module.type = VOTING_TYPE
   }
   else if (id == TREASURY_ID) {
-    Treasury.create(event.params.addr)
+    Treasury.create(newModuleAdress)
     module.type = TREASURY_TYPE
   }
   else if (id == SUBSCRIPTIONS_ID) {
-    Subscriptions.create(event.params.addr)
+    Subscriptions.create(newModuleAdress)
     module.type = SUBSCRIPTIONS_TYPE
 
-    let subscriptionModule = new SubscriptionModule(event.params.addr.toHex())
-    let subscriptions = SubscriptionsContract.bind(event.params.addr)
+    let subscriptionModule = new SubscriptionModule(newModuleAdress.toHex())
+    let subscriptions = SubscriptionsContract.bind(newModuleAdress)
     subscriptionModule.court = event.address.toHex()
     subscriptionModule.currentPeriod = BigInt.fromI32(0)
     subscriptionModule.governorSharePct = BigInt.fromI32(subscriptions.governorSharePct())
-    subscriptionModule.latePaymentPenaltyPct = BigInt.fromI32(subscriptions.latePaymentPenaltyPct())
     subscriptionModule.feeAmount = subscriptions.currentFeeAmount()
     subscriptionModule.feeToken = subscriptions.currentFeeToken()
     subscriptionModule.periodDuration = subscriptions.periodDuration()
-    subscriptionModule.prePaymentPeriods = subscriptions.prePaymentPeriods()
-    subscriptionModule.resumePrePaidPeriods = subscriptions.resumePrePaidPeriods()
-    subscriptionModule.totalDonated = BigInt.fromI32(0)
-    subscriptionModule.totalPaid = BigInt.fromI32(0)
-    subscriptionModule.totalCollected = BigInt.fromI32(0)
-    subscriptionModule.totalGovernorShares = BigInt.fromI32(0)
     subscriptionModule.save()
   }
   else {
     module.type = 'Unknown'
   }
 
+  let moduleAddresses = config.moduleAddresses
+  moduleAddresses.push(newModuleAdress.toHex())
+  config.moduleAddresses = moduleAddresses
+  config.save()
```

Comment:
Isn't possible to push on the config property directly?

Reply:
No, I don’t think so, look at “Updating existing entities” section, at the end, here: https://thegraph.com/docs/assemblyscript-api#api-reference